calico_containers/pycalico/netns.py

```diff
@@ -180,6 +179,32 @@ def remove_ip_from_ns_veth(container_pid, ip, veth_name_ns):
                        "dev", "%(device)s" % veth_name_ns])
 
 
+def provision_veth(endpoint, ns_pid, veth_name_ns):
```

Comment:
We'd previously had a tidy up of the netns class to remove details about endpoints - the idea being that netns provided generic useful helper functions without really needing to understand the various data structures or anything to do with etcd.

I think this might be worth adding as a method to the Endpoint object, so you can just call:

endpoint.provision_veth()